doc/0000-ccl_note/main.tex

```diff
@@ -202,7 +202,11 @@ \section{Functionality}
 \subsection{Physical constants}
 
 \label{sec:constants}
-We have performed a comparison of the physical constants used in \ccl and included dependencies and external sources. See Table \ref{tab:constants} for percent differences of the constants between these sources.
+We have performed a comparison of the physical constants used in \ccl and included dependencies and external sources. See Table \ref{tab:constants} for percent differences of the constants between these sources. Notice that NIST adopts the CODATA 2014 values. Our final choice of constants for \ccl mainly relies on CODATA 2014 in as much as possible, except for ${\rm M}_\odot$, where we adopt the IAU 2015 value, and for the conversion between parsec and meters, where we take the PDG 2013 value.
```

Comment:
"percent differences" -> "percentage differences"

Reply:
A citation to CODATA would be worthwhile here.

Reply:
Review addressed